ambry-commons/src/main/java/com/github/ambry/commons/ServerMetrics.java

```diff
@@ -211,12 +215,14 @@
   public final Counter ttlAlreadyUpdatedError;
   public final Counter ttlUpdateRejectedError;
   public final Counter replicationResponseMessageSizeTooHigh;
+  public final Map<String, Meter> crossColoFetchBytesRate = new HashMap<>();
+  public final Map<String, Meter> crossColoMetadataExchangeBytesRate = new HashMap<>();
```

Comment:
@Arun-LinkedIn these two metrics are what you need to track realtime cross-colo fetch/metadata request bytes rate. Note that, these metrics are emitted on source node, which are more accurate than existing metrics on puller to indicate cross-colo bandwidth usage.